apps/src/p5lab/AnimationPicker/AnimationPickerBody.jsx

```diff
@@ -300,40 +301,11 @@ export default class AnimationPickerBody extends React.Component {
   }
 }
 
-export const UnconnectedAnimationPickerBody = Radium(AnimationPickerBody);
+export const UnconnectedAnimationPickerBody = AnimationPickerBody;
```

Comment:
I don't think you need this now –– you can just do `export const AnimationPickerBody` at the top and call it a day

Reply:
Although that will require a line change in the AnimationPickerBodyTest file, but I think that's okay too

Reply:
@megcrenshaw I'm actually unclear on how this thing that gets imported is different from the default export above. (Maybe that's what you're getting at though - they're the same?)

If that's the case, what was the original value of having a default export (without Radium) and this separate export (with Radium) for the test? Was this component not actually using Radium to begin with?

Reply:
Wow, you're right, this is bizarre.

Usually, we `export UnconnectedComponentName` which isn't hooked up to Radium and/or state, and we use that for testing. And then we `default export ComponentName` with Radium and/or state for other components to consume.

So this is weird because the original code for `UnconnectedAnimationPickerBody` is actually connected to Radium, AND the default export was actually not connected to Radium, AND the component-with-Radium was the one being tested 🤯.

So you're right, it does seem that the components consuming this component were not using Radium. And you're right that at this point the two are the same. More reason to just get rid of this confusion!

Reply:
Updated: https://github.com/code-dot-org/code-dot-org/pull/47478/commits/52921b50c006dd9b9f7432b1ed018ad0b71d237e